jenkins/init.sls

```diff
@@ -14,6 +15,9 @@ jenkins:
     - home: /srv/jenkins
     - shell: /bin/bash
     - makedirs: True
+{% if jenkins.optional_groups %}
+    - optional_groups: {{jenkins.optional_groups}}
```

Comment:
Can you pipe this through the yaml filter to make it a bit more resilient rather than relying on the implicit stringification of python lists being valid YAML?